packages/loader/container-loader/src/connectionManager.ts

```diff
@@ -177,11 +177,11 @@ export class ConnectionManager implements IConnectionManager {
 
     private clientSequenceNumber = 0;
     private clientSequenceNumberObserved = 0;
-    /** Counts the number of noops sent by the client which may not be acked. */
+    /** Counts the number of non-runtime ops sent by the client which may not be acked. */
     private localOpsToIgnore = 0;
 
     /** track clientId used last time when we sent any ops */
-    private lastSubmittedClientId: string | undefined;
+    public lastSubmittedClientId: string | undefined;
```

Comment:
I think this change should be reverted, as we no longer use lastSubmittedClientId